src/modules/75-cf/pages/onboarding/OnboardingSelectedFlag.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import { Layout, Text } from '@harness/uicore'
+import { Color, FontVariation } from '@harness/design-system'
+import { useStrings } from 'framework/strings'
+import type { Feature } from 'services/cf'
+import { IdentifierText } from '@cf/components/IdentifierText/IdentifierText'
+
+export interface CreateAFlagViewProps {
+  flagCreated?: boolean
+  selectedFlag: Feature
+}
+
+export const OnboardingSelectedFlag: React.FC<CreateAFlagViewProps> = ({ flagCreated, selectedFlag }) => {
+  const { getString } = useStrings()
+
+  return (
+    <Layout.Horizontal
+      data-testid="ffOnboardingSelectedFlag"
+      flex={{ justifyContent: 'flex-start', alignItems: 'center' }}
+      spacing="xsmall"
+      padding={{ top: 'small', bottom: 'small' }}
+      margin={{ top: 'medium' }}
+    >
+      {flagCreated ? (
+        <Text font={{ variation: FontVariation.BODY }} color={Color.GREY_600} tag="div">
+          {getString('cf.onboarding.youCreated')}
+        </Text>
+      ) : (
+        <Text font={{ variation: FontVariation.BODY }} color={Color.GREY_600} tag="div">
+          {getString('cf.onboarding.youreUsing')}
+        </Text>
+      )}
+      <Text font={{ variation: FontVariation.BODY }} color={Color.PRIMARY_6} tag="div">
+        {selectedFlag?.name}
+      </Text>
+      <Text font={{ variation: FontVariation.BODY }} color={Color.GREY_600} tag="div">
+        {getString('common.with')}
+      </Text>
```

Comment:
You might be able to simplify this by using one `<Text>` block
```suggestion
      <Text font={{variation: FontVariation.BODY}}> 
        {getString(flagCreated ? 'cf.onboarding.youCreated' : 'cf.onboarding.youreUsing')}
      
        <Text font={{ variation: FontVariation.BODY }} color={Color.PRIMARY_6} inline>
          {selectedFlag?.name}
        </Text>
      
        {getString('common.with')}
      </Text>
```